rules/windows/powershell/powershell_script/posh_ps_wmi_unquoted_service_search.yml

```diff
@@ -1,4 +1,4 @@
-title: WMIC Unquoted Services Path Lookup
+title: WMIC Unquoted Services Path Lookup - Powershell Script Block Text
```

Comment:
`title: WMIC Unquoted Services Path Lookup - PowerShell`
Like the other rules

Reply:
Modified